src/utils/fetcher.ts

```diff
@@ -0,0 +1,5 @@
+export const fetcher = (url: string | null) => {
+  if (url) {
+    return fetch(url).then((res) => res.json())
+  }
+}
```

Comment:
Is there a reason why the argument `url` is nullable? We should not be handling this in a reusable util.

Reply:
this is the type expected by `fetcher` function at `useSWR` hook